telemetry-self-monitor/env-vars

```diff
@@ -0,0 +1,2 @@
+PROMETHEUS_VERSION=2.45.4
```

Comment:
the file is called envs for the other projects, I think the naming is relevant for the prow job